src/rgw/rgw_sal_motr.h

```diff
@@ -40,6 +43,177 @@ class MotrStore;
 #define RGW_MOTR_BUCKET_HD_IDX_NAME   "motr.rgw.bucket.headers"
 //#define RGW_MOTR_BUCKET_ACL_IDX_NAME  "motr.rgw.bucket.acls"
 
+// Implement watch-notify using Motr FDMI.
```

Comment:
The word "Implement" in its imperative form is a bit confusing here. I'd suggest using "Implementation of" instead, or just dropping it.